.storybook/webpack.config.js

```diff
@@ -4,49 +4,63 @@ const transformTemplateForUtilLocation = require('../src/utils/svgToIconTemplate
 
 const UTIL_LOCATION = '../../utils/icons';
 
-module.exports = {
-  resolve: {
+// SVGs will not load properly if we do not remove the default rule before adding our own
+function removeDefaultStorybookSvgRule(config) {
+  const defaultStorybookLoaderRule = config.module.rules.find(rule =>
+    rule.test.test('.svg')
+  );
+
+  const ruleWithoutSvg = defaultStorybookLoaderRule.test
+    .toString()
+    .replace('svg|', '');
+
+  // Removes forward flashes from regexp string before creating new regexp from string
+  defaultStorybookLoaderRule.test = new RegExp(
+    ruleWithoutSvg.substr(1, ruleWithoutSvg.length - 2)
+  );
+}
+
+module.exports = ({ config }) => {
+  removeDefaultStorybookSvgRule(config);
+
+  config.resolve = {
     modules: [path.resolve(__dirname, '..'), 'node_modules'],
-    extensions: ['.js', '.jsx'],
-  },
-  module: {
-    rules: [
-      {
-        test: /\.svg$/,
-        use: [
-          {
-            loader: 'babel-loader',
-            options: require('../babel.config.js'),
-          },
-          {
-            loader: '@svgr/webpack',
-            options: {
-              template: transformTemplateForUtilLocation(UTIL_LOCATION),
-              expandProps: false,
-              babel: false,
-            },
-          },
-        ],
-      },
-      {
-        test: /\.(png|jpe?g|gif)$/,
-        use: [
-          {
-            loader: 'file-loader',
-            options: {},
+    extensions: ['.js', '.jsx', '.md'],
```

Comment:
We have linting errors for `import/extensions`, there's an [option to make an exception for file types](https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/extensions.md#rule-details) that we can use for markdown, or we can resolve the extension [as done here](https://github.com/PocketDerm/radiance-ui/pull/116/files#diff-cade6814f3054b9a5dc17a12188e5048R6).